_posts/2022-03-16-async-sink-base.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "Asynchronous Base Sink"
+date: 2022-03-17 16:00:00
+authors:
+- CrynetLogistics:
+  name: "Zichen Liu"
+  twitter: "CrynetLogistics"
+excerpt: An overview of the new features of the new Async Base Sink and pointers for building your own concrete sink atop
+---
+
+The basic functionalities of sinks in general are quite similar. They batch records according to user defined buffering hints, sign requests, write them to the destination, retry unsuccessful or throttled requests, and participate in checkpointing.
```

Comment:
> They batch records according to user defined buffering hints

I would say this is not a "basic functionalities" of sink, this is a more advanced feature. A basic sink would just send records one by one